app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cd, source_name, copy_name = command.split()
```

Comment:
To perform a safe unpacking independently from the number of elements after split, you can extract exactly the first three elements and neglect others using the following syntaxis:
```suggestion
    cd, source_name, copy_name, *_ = command.split()
```